Traceback (most recent call last):
  File "E:\Phython 3.10\lib\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "E:\Phython 3.10\lib\site-packages\nbclient\client.py", line 1269, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "E:\Phython 3.10\lib\site-packages\nbclient\util.py", line 85, in wrapped
    return just_run(coro(*args, **kwargs))
  File "E:\Phython 3.10\lib\site-packages\nbclient\util.py", line 60, in just_run
    return loop.run_until_complete(coro)
  File "E:\Phython 3.10\lib\asyncio\base_events.py", line 646, in run_until_complete
    return future.result()
  File "E:\Phython 3.10\lib\site-packages\nbclient\client.py", line 707, in async_execute
    await self.async_execute_cell(
  File "E:\Phython 3.10\lib\site-packages\nbclient\client.py", line 1025, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "E:\Phython 3.10\lib\site-packages\nbclient\client.py", line 919, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
def main():

    warnings.filterwarnings('ignore')

    url = "https://putusan3.mahkamahagung.go.id/direktori/index/pengadilan/pn-banyuwangi/kategori/pidana-umum-1"


    end = 108 + 1
    listHasil = []
    start = 51


    startTime = time.time()
    file_hasil = open("/content/drive/MyDrive/Colab Notebooks/Ekstraksi Informasi/hasilListURLPage.txt", "w", encoding='UTF8')

    for i in range(start, end):
        url = f"{url}/page/{i}.html"
        listHasil = getURLfromWeb(url)
        print(f"--- PAGE {i}")

        for listURL in listHasil:
            file_hasil.write(listURL+"\n")

    file_hasil.close()
    endTime = time.time()
    print(listHasil)
    print('Time Processing : ', endTime-startTime, ' Second')

main()
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mFileNotFoundError[0m                         Traceback (most recent call last)
Cell [1;32mIn[3], line 29[0m
[0;32m     26[0m     [38;5;28mprint[39m(listHasil)
[0;32m     27[0m     [38;5;28mprint[39m([38;5;124m'[39m[38;5;124mTime Processing : [39m[38;5;124m'[39m, endTime[38;5;241m-[39mstartTime, [38;5;124m'[39m[38;5;124m Second[39m[38;5;124m'[39m)
[1;32m---> 29[0m [43mmain[49m[43m([49m[43m)[49m

Cell [1;32mIn[3], line 14[0m, in [0;36mmain[1;34m()[0m
[0;32m     10[0m start [38;5;241m=[39m [38;5;241m51[39m
[0;32m     13[0m startTime [38;5;241m=[39m time[38;5;241m.[39mtime()
[1;32m---> 14[0m file_hasil [38;5;241m=[39m [38;5;28;43mopen[39;49m[43m([49m[38;5;124;43m"[39;49m[38;5;124;43m/content/drive/MyDrive/Colab Notebooks/Ekstraksi Informasi/hasilListURLPage.txt[39;49m[38;5;124;43m"[39;49m[43m,[49m[43m [49m[38;5;124;43m"[39;49m[38;5;124;43mw[39;49m[38;5;124;43m"[39;49m[43m,[49m[43m [49m[43mencoding[49m[38;5;241;43m=[39;49m[38;5;124;43m'[39;49m[38;5;124;43mUTF8[39;49m[38;5;124;43m'[39;49m[43m)[49m
[0;32m     16[0m [38;5;28;01mfor[39;00m i [38;5;129;01min[39;00m [38;5;28mrange[39m(start, end):
[0;32m     17[0m     url [38;5;241m=[39m [38;5;124mf[39m[38;5;124m"[39m[38;5;132;01m{[39;00murl[38;5;132;01m}[39;00m[38;5;124m/page/[39m[38;5;132;01m{[39;00mi[38;5;132;01m}[39;00m[38;5;124m.html[39m[38;5;124m"[39m

File [1;32m~\AppData\Roaming\Python\Python310\site-packages\IPython\core\interactiveshell.py:324[0m, in [0;36m_modified_open[1;34m(file, *args, **kwargs)[0m
[0;32m    317[0m [38;5;28;01mif[39;00m file [38;5;129;01min[39;00m {[38;5;241m0[39m, [38;5;241m1[39m, [38;5;241m2[39m}:
[0;32m    318[0m     [38;5;28;01mraise[39;00m [38;5;167;01mValueError[39;00m(
[0;32m    319[0m         [38;5;124mf[39m[38;5;124m"[39m[38;5;124mIPython won[39m[38;5;124m'[39m[38;5;124mt let you open fd=[39m[38;5;132;01m{[39;00mfile[38;5;132;01m}[39;00m[38;5;124m by default [39m[38;5;124m"[39m
[0;32m    320[0m         [38;5;124m"[39m[38;5;124mas it is likely to crash IPython. If you know what you are doing, [39m[38;5;124m"[39m
[0;32m    321[0m         [38;5;124m"[39m[38;5;124myou can use builtins[39m[38;5;124m'[39m[38;5;124m open.[39m[38;5;124m"[39m
[0;32m    322[0m     )
[1;32m--> 324[0m [38;5;28;01mreturn[39;00m io_open(file, [38;5;241m*[39margs, [38;5;241m*[39m[38;5;241m*[39mkwargs)

[1;31mFileNotFoundError[0m: [Errno 2] No such file or directory: '/content/drive/MyDrive/Colab Notebooks/Ekstraksi Informasi/hasilListURLPage.txt'
FileNotFoundError: [Errno 2] No such file or directory: '/content/drive/MyDrive/Colab Notebooks/Ekstraksi Informasi/hasilListURLPage.txt'

